docs/packaging/msix-packaging-tool.md

```diff
@@ -69,15 +69,17 @@ on the size of the package.
     * Add `previewsecurity2` to `IgnorableNamespaces` at the end of the `<Package>` element
 
     * In `<Dependencies>` change `TargetDeviceFamily` to
-    `<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.25229.0" MaxVersionTested="10.0.25229.0" />`
+    `<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.25357.0" MaxVersionTested="10.0.25357.0" />`
+
+        * **Note**: Not all features are available in the minimun build, check out the [release notes](../../relnotes/windows-release-notes.md) for more detailed information.
 
     * In `<Application>` replace any existing entrypoint/trustlevel/runtimebehavior with
     `uap10:TrustLevel="appContainer" previewsecurity2:RuntimeBehavior="appSilo"`
 
-    * **Note**: By default MPT, will automatically add `<rescap:Capability name="runFullTrust">` to
-    `<Capabilities>` due to the app being a packaged Win32. This should be removed unless
-    the app has other manifested extensions which can affect the user global state, such as
-    `comServer` or `FirewallRules`, since those require the `runFullTrust` capability.
+        * **Note**: By default MPT, will automatically add `<rescap:Capability name="runFullTrust">` to

```

Comment:
Will this render like this note belongs to the bullet point above(`<Application>` one)? I think it works well with the previous indentation, thoughts?

Also, is it by default MPT, or by default, MPT will?

Reply:
This will render the notes a sub-bullet 
![image](https://github.com/microsoft/win32-app-isolation/assets/128075585/467209ad-71fd-4073-8d32-88e1c5433ad2)

Reply:
Yeah this is what I meant. The note above makes sense because the note is for minimum OS version, so it should be a sub-bullet of the `Dependencies` bullet. However, the note below seems to be part of the big picture(manifests in general), not part of the `Application` section. So would it make more sense to keep it as the same level bullet as the others? Basically do not add the extra indentations in md files for the latter note.